code/controllers/configuration.dm

```diff
@@ -904,7 +907,12 @@ var/global/list/gamemode_cache = list()
 					config.use_loyalty_implants = 1
 				if("dexterity_malus_brainloss_threshold")
 					config.dex_malus_brainloss_threshold = text2num(value)
-
+				if("grant_default_darksight")
+					config.grant_default_darksight = TRUE
+				if("default_darksight_range")
+					config.default_darksight_range = clamp(text2num(value), 0, 1)
```

Comment:
This is 2 by default, but if set in the config, it's clamped between 0 and 1. That seems incorrect. Looks like it should probably just have a minimum of 0.
```suggestion
					config.default_darksight_range = max(text2num(value), 0)
```